kmir/k-src/mir-rvalue.md

```diff
@@ -229,9 +229,40 @@ Evaluate a syntactic `RValue` into a semantics `RValueResult`. Inspired by [eval
   rule evalConstantValue(VALUE:StringLiteral)   => StringLitertal2String(VALUE)
   rule evalConstantValue(( ))                   => Unit
   rule evalConstantValue(VALUE:Bool)            => VALUE
+  rule evalConstantValue(VALUE:ConstEnumConstructor)  => evalPrimitiveBound(VALUE)
 //  rule evalConstantValue(_VALUE)              => "Error: evalConstantValue --- unsupported ConstantValue" [owise]
 ```
 
+#### Primitive type bounds TODO: usize depends on architecture, which we currently do not handle
```

Comment:
The changes here all seem fine to me; let me know if you would like any pointers or advice from our experience with the C Semantics when you come around to implementing the architecture-specific bits.